assets/sass/patterns/organisms/content-header.scss

```diff
@@ -25,6 +24,49 @@
 
 }
 
+// Content Header Grid Articles
+
+.content-header-grid {
+  padding: 0 24px;
```

Comment:
I see the `24px` value used a lot in these changes. Is there not a variable similar to `$content-header-padding` that can be used? If not, would it not be wise to add one?

Reply:
I have set up $site-spacing-bottom in the variables sass file.